### PR TITLE
Fixes 216: a tag is added to img tag for make it clickable

### LINKED ISSUE #216: Button on the themed emails are not clickable
### This is a
- [x] Bug report
- [ ] Feature request

### Current behaviour

Nothing happens when clicking buttons in the themed email

### Steps to reproduce
1.
2.
3.

### PR DESCRIPTION
**Pull request by sch00lb0y**
## Description

I have added a tag to existing img tag to make it clickable
## Related Issue

close #216 
## Motivation and Context

By making this change now users can directly click the icon as well the below text to redirect
## Types of changes

- [x] Bug fix 
- [ ] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

**Comment by pri22296** [comment]
How has this been tested?

**Comment by sch00lb0y** [comment]
@pri22296 here is rendered email http://heavy-quarter.surge.sh/

**Comment by sch00lb0y** [comment]
@imujjwal96 please check this

File: views/template/email.jade [hunk 1]
```diff
@@ -90,7 +90,8 @@ html(xmlns='http://www.w3.org/1999/xhtml', xmlns:v='urn:schemas-microsoft-com:vm
                             tbody
                               tr
                                 td(style='width:85px;')
-                                  img(alt='', title='', height='auto', src='http://res.cloudinary.com/template-gdg/image/upload/v1498552339/play_cuqe89.png', style='border:none;border-radius:0px;display:block;font-size:13px;outline:none;text-decoration:none;width:100%;height:auto;', width='85')
+                                  a(href="#{previewURL}")
```
Comment (imujjwal96) [inline views/template/email.jade]: use target = _blank

Comment (sch00lb0y) [inline views/template/email.jade]: made changes

**Pull request merged**